docs/package.json

```diff
@@ -14,16 +14,17 @@
     "@ant-design/colors": "^4.0.0",
     "@ant-design/icons-svg": "^4.0.0",
     "classnames": "^2.2.6",
+    "compression": "^1.7.1",
     "css-animation": "^1.5.0",
     "lodash": "^4.17.15",
-    "raf": "^3.4.1",
-    "compression": "^1.7.1",
     "polka": "next",
     "postcss": "^7.0.27",
     "postcss-load-config": "^2.1.0",
     "prism-svelte": "^0.4.6",
     "prismjs": "^1.20.0",
-    "sirv": "^0.4.0"
+    "raf": "^3.4.1",
+    "sirv": "^0.4.0",
+    "to-style": "^1.3.3"
```

Comment:
This is needed for a followup PR, I accidentally added these in this one but I think it is fine.